codegen/src/main/java/software/amazon/awssdk/codegen/internal/Jackson.java

```diff
@@ -43,6 +43,8 @@
         MAPPER.configure(JsonParser.Feature.ALLOW_COMMENTS, true);
         // TODO: Un comment this once we know for sure, we capture everything in C2j model.
         MAPPER.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+        // TODO: Risky to enable for all deserialiation?
+        MAPPER.enable(DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY);
```

Comment:
What's this for?

Reply:
In paginators.json, the values for input_token, output_token and result_key can be a single value or a list of values. 
Example:
 "result_key": ["Folders", "Documents"]
 "result_key": "Users"

I am representing these fields as List in PaginatorDefinition class. When deserializing the json file into PaginatorDefinition class, this property has to be set so mapper knows to accept a single value into a list.